content/run.md

```diff
@@ -152,7 +152,7 @@ input argument variables:
     
 First, you need to change the global input data directory variable inside the script (standard global CTSM dataset) as below:
 
-    dir_inputdata='/cluster/shared/noresm/inputdata/'
+    dir_inputdata='/cluster/shared/noresm/inputdata/'  (somelines after region.create-tag())
```

Comment:
Why add this? You can easily look for the dir_inputdata variable in the code.

Reply:
It seems this is necessary considering Gunnar's experience. There is another place of dir_inputdata for site run, it does not work if you change there.

Reply:
Understood. But could you give perhaps give a line number +/-? The text should be moved and included in line 153 instead. "First, you need to change the variable pointing to the CTSM input data directory on your system. The variable can be found xx lines after the function `region.create-tag()`."

Remark: If you set f.e. a CTSM_INPUTDATA environment variable, the code could be changed to `dir_inputdata = os.environ['CTSM_INPUTDATA']`. Like this, you can easily switch the input directory without changing the script.